addons/binding/org.openhab.binding.vitotronic/ESH-INF/thing/thing-types.xml

```diff
@@ -15,12 +15,41 @@
 
         <channels>
             <channel id="systemtime" typeId="systemtime"/>
-            <channel id="outside_temp" typeId="outside_temp"/>
-            <channel id="boiler_temp" typeId="boiler_temp"/>   
-            <channel id="flowuprating" typeId="flowuprating"/>   
+            <channel id="outside_temp" typeId="temperature_ro">

```

Comment:
I reworked the channel types. Please check it with the next commit.
But I still don´t understand the advantage. You have more experience - do you have a usecase?
I was happy having the file a lot smaller ...